parser/src/main/java/com/linkedin/avroutil1/parser/avsc/AvscParser.java

```diff
@@ -400,10 +400,11 @@ private AvroNamedSchema parseNamedSchema(
                             }
                             LiteralOrIssue defaultValueOrIssue = parseLiteral(fieldDefaultValueNode, defaultValueExpectedSchema, fieldName.getValue(), context);
                             if (defaultValueOrIssue.getIssue() == null) {
-                                //TODO - allow parsing default values that are branch != 0 (and add an issue)
```

Comment:
default values for branches != 0 are not allowed per the avro spec

Reply:
true, but could have parsed the default value itself for any errors, in theory.